dashboard/src/shared/types.ts

```diff
@@ -77,9 +75,8 @@ export interface IChart {
 }
 
 export interface IReceiveChartsActionPayload {
-  items: AvailablePackageSummary[];
+  response: GetAvailablePackageSummariesResponse;
   page: number;
-  nextPageToken: string;
 }
```

Comment:
I've simplified this payload to avoid having unnecessary extra fields (especially, now we have autogenerated models)